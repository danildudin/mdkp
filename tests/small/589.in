11 5 0
63 79 24 76 25 28 21 60 34 83 40 
38 43 83 7 24 50 77 24 7 88 97 
24 36 32 20 21 45 35 4 5 17 48 
88 41 8 45 71 94 10 61 42 100 86 
93 89 60 20 87 89 16 21 79 39 4 
40 16 42 36 98 70 44 17 100 87 80 
322 48 308 599 530 
