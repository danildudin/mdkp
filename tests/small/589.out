253
4
3 7 8 9 
